Core/Security/VOMS.py

```diff
@@ -193,10 +193,11 @@ def getVOMSESLocation(self):
     requiredFilePerms = stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH
     # 777
     allPerms = stat.S_IRWXU | stat.S_IRWXG | stat.S_IRWXO
-    vomsesPaths = []
-    if 'DIRAC_VOMSES' in os.environ:
-      vomsesPaths.append(os.environ['DIRAC_VOMSES'])
-    vomsesPaths.append(os.path.join(rootPath, "etc", "grid-security", "vomses"))
+    if 'X509_VOMSES' not in os.environ:
+      return S_ERROR("The env variable X509_VOMSES is not set. "
```

Comment:
This function doesn't normally return S_OK, so wherever this is called is going to fail

Reply:
would you rather see that I an exception?

Reply:
I believe at the moment you would already get an exception when the return value is used and is a dict instead of a string 😀

Reply:
so let's make it exception directly